base/util/src/main/java/com/netscape/cmsutil/crypto/CryptoUtil.java

```diff
@@ -1431,51 +1427,47 @@ public static String getSKIString(X509CertImpl cert) throws IOException {
     /*
      * get extension from CRMF cert request (CertTemplate)
      */
-    public static org.mozilla.jss.netscape.security.x509.Extension getExtensionFromCertTemplate(CertTemplate certTemplate, ObjectIdentifier csOID) {
+    public static org.mozilla.jss.netscape.security.x509.Extension getExtensionFromCertTemplate(
+            CertTemplate certTemplate, ObjectIdentifier csOID) {
         //ObjectIdentifier csOID = PKIXExtensions.SubjectKey_Id;
-        OBJECT_IDENTIFIER jssOID =
-                new OBJECT_IDENTIFIER(csOID.toString());
-/*
+        OBJECT_IDENTIFIER jssOID = new OBJECT_IDENTIFIER(csOID.toString());
+        /*
         return getExtensionFromCertTemplate(certTemplate, jssOID);
-    }
-    public static org.mozilla.jss.netscape.security.x509.Extension getExtensionFromCertTemplate(CertTemplate certTemplate, org.mozilla.jss.asn1.OBJECT_IDENTIFIER jssOID) {
-*/
+            }
+            public static org.mozilla.jss.netscape.security.x509.Extension getExtensionFromCertTemplate(CertTemplate certTemplate, org.mozilla.jss.asn1.OBJECT_IDENTIFIER jssOID) {
```

Comment:
I think the indentation will make it more confusing. Should we just remove the whole comment?

Reply:
Yes, I agree.